src/map/skill.cpp

```diff
@@ -18950,7 +18950,7 @@ int skill_vfcastfix(struct block_list *bl, double time, uint16 skill_id, uint16
 		}
 		for (const auto &it : sd->skillvarcast) {
 			if (it.id == skill_id) { // bonus2 bSkillVariableCast
-				time += it.val;
+				time = max(0, time + it.val);
```

Comment:
The range is checked later, do these 2 changes change anything?
```
time = max((int)time, 0) + (1 - (float)min(fixcast_r, 100) / 100) * max(fixed, 0); //Underflow checking/capping
```

Reply:
Seem like it's no longer problem in latest commits. I will close this PR.

Thank you for reviewed!